app/V1Module/presenters/GroupsPresenter.php

```diff
@@ -166,6 +166,7 @@ public function actionDefault(
      * @Param(type="post", name="externalId", required=false, description="An informative, human readable identifier of the group")
      * @Param(type="post", name="parentGroupId", validation="string:36", required=false, description="Identifier of the parent group (if none is given, a top-level group is created)")
      * @Param(type="post", name="publicStats", validation="bool", required=false, description="Should students be able to see each other's results?")
+     * @Param(type="post", name="detaining", validation="bool", description="Can students leave the group on their own?")
```

Comment:
This seems a bit reversed... If I answer true to your question than the group will not be detaining...

Reply:
Right...